client/src/pages/Login/index.js

```diff
@@ -1,6 +1,90 @@
+import { LockOutlined, MailOutlined } from '@ant-design/icons';
+import { Button, Form, Input } from 'antd';
+import PropTypes from 'prop-types';
 import React from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import loginImg from '../../assets/loginImg.png';
+import { AuthConsumer } from '../../context';
 import './style.css';
 
-const Login = () => <div className="Login" />;
+class Login extends React.Component {
+  state = {
+    error: false,
+  };
+
+  handleSubmit = (values, getAuth) => {
+    const { email, password } = values;
+    fetch('/api/v1/login', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        email,
+        password,
+      }),
+    }).then((res) => {
+      this.setState({
+        error: !res.ok,
+      });
+
+      if (res.ok) {
+        getAuth();
+      }
+    });
+  };
+
+  render() {
+    const { error } = this.state;
+    const { previousLocation } = this.props;
+
+    const errorMessage = error ? (
+      <p className="error-message">Please correct your email or password</p>
+    ) : null;
+    return (
+      <AuthConsumer>
+        {({ logged, getAuth }) => (
```

Comment:
you can use contextType instead of the consumer for a bit friendlier looking code